src/lib/y2security/clients/security_policy_proposal.rb

```diff
@@ -309,6 +333,15 @@ def links_for_rule(rule)
           ]
         end
 
+        # Possible links to set the SCAP action
+        #
+        # @return [Array<String>]
+        def links_for_scap_actions
```

Comment:
NP: I would try to keep the same name style as the rest of methods, e.g., `#scap_actions_links`.

Reply:
I thought the style was `links_*`. We have `links_for_rule` and `links_for_policy` already. See https://github.com/yast/yast-security/blob/190360d82b2d8b5efa24cd42bf139e7afe510abe/src/lib/y2security/clients/security_policy_proposal.rb#L132-L134 for its usage.